src/app/globals.css

```diff
@@ -34,6 +34,7 @@ body {
   font-family: var(--font-family-primary);
   background: var(--color-shade-first);
   color: var(--color-secondary);
+  background-image: url('backgroundimage.jpg');
```

Comment:
I would rename the file to `background.jpg`. All `.jpg` files are `images`.

Reply:
okay. done 👍